indra/llwebrtc/llwebrtc.cpp

```diff
@@ -959,7 +965,7 @@ void LLWebRTCPeerConnectionImpl::setMute(bool mute)
     mMute = mute;
     mute |= mWebRTCImpl->isCaptureNoDevice();
     mWebRTCImpl->PostSignalingTask(
-        [&]()
```

Comment:
Passing in mute by reference means mute could be altered after leaving the function call, before entering the lambda.  This was resulting in users being able to hear another user even when they're muted.